Dockerfile

```diff
@@ -66,7 +66,7 @@ ENV GTEST_COLOR=1
 COPY --chown=axom:axom . /home/axom/workspace
 WORKDIR /home/axom/workspace
 ENV HCC_AMDGPU_TARGET=gfx900
-RUN mkdir build && cd build && cmake -DROCM_ROOT_DIR=/opt/rocm/include -DENABLE_DEVELOPER_DEFAULTS=On -DENABLE_HIP=On ..
+RUN mkdir build && cd build && cmake -DROCM_ROOT_DIR=/opt/rocm/include -DHIP_RUNTIME_INCLUDE_DIRS="/opt/rocm/include;/opt/rocm/hip/include" -DENABLE_DEVELOPER_DEFAULTS=On -DENABLE_HIP=On ..
```

Comment:
Are we not running tests with hip (or apparently nvcc)?

Reply:
Not on Azure.

Reply:
duh